### DISCUSSION TITLE
can we have a nerdctld(daemon process) to support a restful api for other client to manage container?

### DISCUSSION THREAD

**Discussion by ningmingxiao** [containerd/nerdctl#958] [Q&A] [answered]
### What is the problem you're trying to solve

nerdctl doesn't have api

### Describe the solution you'd like

we  should have a nerdctld(daemon process) to support a restful api for other client to manage container

**Comment by AkihiroSuda** [comment, marked as answer]
@afbjorklund has been experimenting this in his repo
https://github.com/afbjorklund/nerdctld

**Reply by afbjorklund** [reply]
It was more of a hack, I think that if you want Docker API support for containerd the primary option should be to keep running Docker Engine...

But if you are determined to replace "docker" CLI with `nerdctl`, this would be a way to replace (parts of) "dockerd" API with `nerdctld`.

**Comment by huapox** [comment]
+1
